CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Fix: return error to client if expresson or multientity plugins gets error
```

Comment:
Should https://github.com/telefonicaid/iotagent-node-lib/issues/1175 or https://github.com/telefonicaid/iotagent-node-lib/issues/1223 or both be mentioned here?

Reply:
Just related, but this PR does not fix them

Reply:
NTC